projects_test.go

```diff
@@ -44,3 +44,55 @@ func TestRemoveProject(t *testing.T) {
 	assert.Equal(t, err, nil)
 	assert.Equal(t, result, true)
 }
+
+func TestListVariables(t *testing.T) {
+	ts, gitlab := Stub("stubs/projects/variables/index.json")
+
+	variables, err := gitlab.ProjectVariables("1")
+
+	assert.Equal(t, err, nil)
+	assert.Equal(t, len(variables), 2)
+	defer ts.Close()
+}
+
+func TestGetVariable(t *testing.T) {
+	ts, gitlab := Stub("stubs/projects/variables/show.json")
+
+	_, err := gitlab.ProjectVariable("1", "Somekey")
+
+	assert.Equal(t, err, nil)
```

Comment:
You should use ``assert.NoError``  ;) 
You should also test the value returned by the function